presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -450,6 +450,27 @@ Column Type   Collectible Statistics
 Automatic column level statistics collection on write is controlled by
 the ``collect-column-statistics-on-write`` catalog session property.
 
+.. _hive_analyze_label:
+
+Collecting table and column statistics
+--------------------------------------
+
+The Hive connector supports collection of table and partition statistics
+via Presto :doc:`/sql/analyze` command. When analyzing partitioned table it is possible
+to specify which partitions needs to be analyzed via optional ``partitions`` analyze
+property. ``partitions`` property is of ``array(array(varchar))`` format where
```

Comment:
This needs a better explanation. Something along the lines of each element of the array being an array containing the values of the partition keys in the order in which they are declared in the table schema.